### ISSUE TITLE
Improve image alt texts

### ISSUE THREAD

**Issue by rain2o** [Myzwer/foothillschurch#36]
https://github.com/Myzwer/foothillschurch/blob/effc5bbf81497ae99f0561ab7849412fe9272bfd/contact.php#L23

I'm only linking one example, but this goes for all images.

I've noticed you've provided alt text to your images, which I appreciate. However, some of them shouldn't actually have alt text, and some of them could be improved. 

Some basic rules of thumb:
* If the image is decorative only, meaning it doesn't provide any additional information to the user, then the alt text should be empty `<img alt="" />`
* If the image provides information that is useful to the user, the alt text should describe the content being portrayed.
* Don't use words like "image" or "photo" in the alt text. Screen readers tell the users it's an image, followed by the alt text.

You can kind of think of it like if someone wanted a text-only version of the website without any design. What would you give the user in place of the image, if anything? 

For more, this is a good article talking about it: https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices

**Comment by Myzwer** [comment]
This one was a DOOZY. Replaced every image on the site (in the code) with an object array to load in the alt message, then wrote alt messages for every image on the site in WP backend. But it's done.
